lib/gs-plugin-loader.c

```diff
@@ -2978,10 +2946,9 @@ gs_plugin_loader_network_changed_cb (GNetworkMonitor *monitor,
 		g_autoptr(GsAppList) queue = NULL;
 		g_mutex_lock (&priv->pending_apps_mutex);
 		queue = gs_app_list_new ();
-		for (guint i = 0; i < priv->pending_apps->len; i++) {
-			GsApp *app = g_ptr_array_index (priv->pending_apps, i);
-			if (gs_app_get_state (app) == AS_APP_STATE_QUEUED_FOR_INSTALL)
```

Comment:
Again, is it safe to skip the check for AS_APP_STATE_QUEUED_FOR_INSTALL?

Reply:
And the question still applies here, too.

Reply:
See the lousy answer to the other question.